frontend/packages/kubevirt-plugin/src/components/vms/vm-details.scss

```
@@ -0,0 +1,4 @@
+.kubevirt-vm-details__pending-changes-class-alert {
```

I've asked before about naming convention and if we can shorten them. What about this:
```suggestion
.kv-vm__pending-changes-alert {
```

---

+1